projects/opendr_ws/src/perception/scripts/point_cloud_dataset.py

```diff
@@ -53,6 +53,8 @@ def start(self):
             message = self.bridge.to_ros_point_cloud(
                 point_cloud
             )
+            message.header.frame_id = "velodyne"
```

Comment:
Thank you for your contribution. We can rename the `frame_id` to something more generic.

```suggestion
            message.header.frame_id = "lidar_dataset"
```